internal/provisioner/helm/controllers/bundledeployment_controller.go

```diff
@@ -178,7 +189,13 @@ func (r *BundleDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, err
 	}
 
-	rel, state, err := r.getReleaseState(cl, bd, chrt, values)
+	post := &postrenderer{
+		labels: map[string]string{
+			util.CoreOwnerKindKey: "BundleDeployment",
```

Comment:
nit: I think we have a const variable somewhere for the BD GVK.

Reply:
It doesn't seem defined.